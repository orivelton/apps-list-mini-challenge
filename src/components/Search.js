import React from 'react';

const Search = ({valueSearch, handleSearchTerm, setIsCategories}) => {
  return (
    <header>
      <input 
        type="text" 
        placeholder="Search by App" 
        value={valueSearch} 
        onChange={({target: { value}}) => handleSearchTerm(value, true)}
      />
    </header>
  )
};

export default Search;
