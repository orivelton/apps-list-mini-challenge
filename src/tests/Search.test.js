import React from 'react';
import renderer from 'react-test-renderer';
import Search from '../components/Search';

test('renders correctly', () => {
  const tree = renderer.create(<Search />).toJSON();
  expect(tree).toMatchSnapshot();
});
