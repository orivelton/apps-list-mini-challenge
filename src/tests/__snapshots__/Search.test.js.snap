// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header>
  <input
    onChange={[Function]}
    placeholder="Search by App"
    type="text"
  />
</header>
`;
